about.html

```diff
@@ -0,0 +1,174 @@
+        </nav>
+        </header>
+        <!--End of Header section-->
+    </header>
+    <!--Header section-->
+    <section class="mainS"></section>
+           <div class="main-header">
+            <p class="hello">"Hello! Am a WFH"</p>
+                   <h1 class="header-text">
+                        REMOTE DEV'S ANNUAL GLOBAL &nbsp;SUMMIT 2022 
+          </h1>
+          <p class="desc"> The after month of covid 19 saw an over whelming growth of the demand for remote workers
+            Previously a very slow sector experienced a sharp demand as companies had also come to appreciate thie benefits associtaed. the initially 
+            unknown event now gathers all works of life with a turn up of thousands each year. For the first Time, its in AFRICA , Kampala, Uganda in E.Africa this OCTOBER</p>
+             <p class="desc2">Please contact @www.MunyonyoCommonwealth esort.ug, Along Lake Victoria Shores, @www.TripeAdvisors.com, www.Eaafricansafari.com</p>
+             </div>
+        <div class="middle">
+             <div class="mid-label">
+              <!-- <div class="footer-head ">
+                <a href="#"> 
+                  <img class="logo-img" src="images/proLogo.png" alt="logo"/></a>
+                   <p>
+                Ⓒ 2022 WFH Uganda <br />
+                All Rights Reserved
+              </p>
+              </div> -->
+            
+                  <!-- <h3>REMOTE <br>GLOBAL <br>SUMMIT</h3> -->
+                  
+             <h3 class="pearl">@ Pearl</h3>
+             <img src="images/Equator.jpg" alt="The Equater" class="pearl-img" title="@ the Equater">
+             <img src="images/Pearl-of-Africa-Hotel.jpg" alt="Speke Resort Hotel" class="pearl-img" title="Speke Resort Hotel">
+            
+          </div>
+          </div>
+    <!--End of Header section-->
+
+    <!--Middle section-->
+    <div class="middle">
+              <h2 class="date">2022.10.15(MON ~ 17(WED)</h2>
+              <h3>REMOTE DEV's GLOBAL SUMMIT ~2022</h3>
+              <p>
+                Kampala, Uganda (The Pearl of AFRICA) <br />
+                Its has never been better
+              </p>
+            
+            
+            </div>
+    <!--End of Middle section-->
+
+    <!--Past section-->
+    <section class="past">
+                <h2 class="past-header">See the past Global Summits</h2> <hr>
+                <p class="desc">Take a look at the last two global summits which took place in Buenos Aires and in warsaw</p>
+               
+                <div class="past-img">
+                
+                <div class="past1"></div><div class="center">
+                  <h4 class="center-year">
+                    2020</h4><p>Remote Global summit 2020 warsaw</p></div>
+                  <div class="past2"></div>
+                  <div class="center1">
+                    <h4 class="center-year">2021</h4><p>Remote Global summit 2021 Buenos Aires</p></div>
+              </div>
+    </section>
```

Comment:
- [ ] Kindly try to match the guidelines as much as possible in the 'past events' section.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190241472-b299108d-6f11-4467-affa-96629a98cb83.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/190241642-9e0fd2fa-0db0-4807-8d24-33aa6eeb0937.png)